docs/advanced-topics/wes/wes-cwl-agc-tutorial.rst

```diff
@@ -0,0 +1,255 @@
+
+6. To verify that the Dockstore CLI is communicating with the AGC infrastructure, list the WES server info. A JSON response will be printed
+to your terminal with the server's configuration.
+
+    .. code:: text
+
+        dockstore workflow wes service-info
+
+.. note::
+    At this point, the AGC infrastructure is deployed and the Dockstore CLI has been configured.
+
+    The AGC context and Dockstore configuration file do not need to be modified for the remainder of these examples, and will continue to function until the resources are modified and/or destroyed.
+
+
+Words Workflow
+--------------
+The Dockstore entry associated with this workflow can be found here `words <https://dockstore.org/workflows/github.com/dockstore-testing/tooltester-wes-testing/words:stable-version-for-testing-v4?tab=info>`_.
+
+
+
+    *main.cwl*
+
+    .. code:: text
+
+        cwlVersion: v1.0
+        class: Workflow
+        requirements:
+          ScatterFeatureRequirement: {}
+        inputs:
+          words: File
+          vowels: string[]
+        outputs:
+          summaryFile:
+            type: File
+            outputSource: sumWords/summaryFile
+
+        steps:
+          countWordsWithLetter:
+            scatter: vowel
+            in:
+              words: words
+              vowel: vowels
+            out: [countFile]
+            run:
+              class: CommandLineTool
+              baseCommand: grep
+              inputs:
+                words: File
+                vowel: string
+              arguments:
+                - $(inputs.vowel)
+                - $(inputs.words.path)
+                - --count
+              outputs:
+                countFile:
+                  type: stdout
+              stdout: count.txt
+          sumWords:
+            in:
+              countFiles: [countWordsWithLetter/countFile]
+            out: [summaryFile]
+            run:
+              class: CommandLineTool
+              baseCommand: ["awk", "{ sum += $1 } END { print sum }"]
+              inputs:
+                countFiles:
+                  type: File[]
+                  inputBinding:
+                    position: 1
+              outputs:
+                summaryFile:
+                  type: stdout
+              stdout: summary.txt
+
+1. This workflows takes a file, and an array of strings as an input.  Create a file named ``input.json`` in your working directory with the contents:
+
+    *input.json*
+
+    .. code:: text
+
+            {
+              "words": {
+                "class": "File",
+                "path": "https://raw.githubusercontent.com/dockstore-testing/tooltester-wes-testing/178957c33c37ce5b91e2c973c1f5dd6870c31b6a/words/mieliestronk-words.txt"
+              },
+              "vowels": ["a","e","i","o","u"]
+            }
+
+2. Since this workflow is publicly posted on `Dockstore.org <https://dockstore.org/>`_, we can quickly launch it by passing the Dockstore CLI the entry and input files.
```

Comment:
It would be nice to include a link to the workflow on Dockstore: https://dockstore.org/workflows/github.com/dockstore-testing/tooltester-wes-testing/words:stable-version-for-testing-v4?tab=info

Reply:
Good idea. Fixed!